services/ui-src/src/utils/api/requestMethods/getTemplateUrl.ts

```diff
@@ -1,12 +1,16 @@
-import { API } from "aws-amplify";
+import { get } from "aws-amplify/api";
 import { getRequestHeaders } from "./getRequestHeaders";
 
 export async function getSignedTemplateUrl(templateName: string) {
   const requestHeaders = await getRequestHeaders();
-  const request = {
+  const options = {
     headers: { ...requestHeaders },
   };
 
-  const response = await API.get("mcr", `/templates/${templateName}`, request);
-  return response;
+  const { body } = await get({
+    apiName: "mcr",
+    path: `/templates/${templateName}`,
+    options,
+  }).response;
+  return (await body.json()) as string;
```

Comment:
does body.json() already return a string?

Reply:
Oh, interesting!  Yeah, it looks like that's necessary.  The only alternative I think of is: 
`String(await body.json())` that's just style preference.